### ISSUE TITLE
Regarding Changing Font

### ISSUE THREAD

**Issue by devenpawar18** [weidongjian/androidWheelView#58]
@weidongjian Could you add a way to change font of the scroll items?

Thanks

**Comment by weidongjian** [comment]
resolved 
https://github.com/weidongjian/androidWheelView/releases/tag/0.2.0

**Comment by devenpawar18** [comment]
@weidongjian This doesn't seem to be working. I tried following:
```
val typeface = ResourcesCompat.getFont(context, R.font.my_font)
loopView.setTypeface(typeface)
```

Could you tell me If I'm doing anything wrong?

**Comment by weidongjian** [comment]
@devenpawar18 problem resolved at 0.2.1 
https://github.com/weidongjian/androidWheelView/releases
